src/bookofnim/deepdives/asyncPar.nim

```diff
@@ -5,6 +5,7 @@
 
 ##[
 ## TLDR
+- additional info for nimv2 in asyncParV2.nim
```

Comment:
remove this or we'll have to put this in every file
there should be a single comment at the top of bookofnim.nim